### PR TITLE
docs: remove docs

### PR DESCRIPTION
**Pull request by ChristianDarr-personal**
**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)N/A
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?) N/A
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>
https://github.com/edgexfoundry/edgex-docs/pull/1058

## Testing Instructions
N/A

## New Dependency Instructions (If applicable)
N/A

### CODE REVIEW

**Review by lenny-goodell — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -7,53 +7,8 @@
...
-## References
-- EdgeX Foundry Project Wiki: https://wiki.edgexfoundry.org/
-- EdgeX Source Code: https://github.com/edgexfoundry
-- Edgex Developer Guide: https://docs.edgexfoundry.org/2.1/
-- Docker Repos
-   - Docker https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository
-   - Docker Compose https://docs.docker.com/compose/install/#install-compose
-
+## General
+Visit the [EdgeX documentation page](https://docs.edgexfoundry.org/3.0/microservices/device/supported/device-usb-camera/General/) to learn more about the latest release.
```
Comment (ajcasagrande) [inline README.md]: Should the url point to 3.0 or `latest`?

**Review by ajcasagrande — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (lenny-goodell) [reply README.md]: > Should the url point to 3.0 or `latest`?

`latest` (will be odd, until the 3.0 docs are released)

**Review by lenny-goodell — APPROVED** [review]
LGTM

**Pull request merged**